### PR TITLE
bug-fix-3.6/internal-issue-#704

### PR DESCRIPTION
**Pull request by gnusi**
backport of https://github.com/arangodb/backlog/issues/704

### CODE REVIEW

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9845/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9847/

File: 3rdParty/iresearch/core/formats/formats_burst_trie.cpp [hunk 1]
```diff
@@ -421,62 +421,112 @@ class block_iterator : util::noncopyable {
...
+  int16_t next_label() const noexcept { return next_label_; }
+  uint64_t sub_count() const noexcept { return sub_count_; }
+  uint64_t start() const noexcept { return start_; }
+  bool end() const noexcept { return cur_ent_ == ent_count_; }
+
+  template<typename Reader>
+  SeekResult scan_to_term(const bytes_ref& term, Reader& reader) {
+    assert(!dirty_);
+
+    if (cur_ent_ == ent_count_) {
```
Comment (Dronplane) [inline 3rdParty/iresearch/core/formats/formats_burst_trie.cpp:476]: This block was removed in upstream PR. Is this correct ?

Comment (Dronplane) [reply 3rdParty/iresearch/core/formats/formats_burst_trie.cpp:476]: https://github.com/iresearch-toolkit/iresearch/pull/183/files

**Comment by jsteemann** [comment]
Quickly discussed with @gnusi, and the decision was to close the PR for now.

**Pull request closed without merging**